src/gui/editors/AutomationEditor.cpp

```diff
@@ -1099,8 +1098,8 @@ inline void AutomationEditor::drawCross( QPainter & p )
 	float scaledLevel = m_pattern->firstObject()->scaledValue( level );
 
 	// Limit the scaled-level tooltip to the grid
-	if( mouse_pos.x() > LEFT_MARGIN &&
-		mouse_pos.x() < right_margin &&
+	if( mouse_pos.x() > VALUES_WIDTH - 1 && // left_margin
```

Comment:
I think `VALUES_WIDTH` would be better, because it matches the limit area to where you can edit points.